data/yaml/venues/delite.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: DELITE
+name: The First Workshop on Language-driven Deliberation Technology (DELITE2024)
```

Comment:
Remove first and the acronym from the name

Reply:
Done.